### PR TITLE
udpdate SLO doc images

### PR DESCRIPTION
**Pull request by roxanne-moslehi**
### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: static/images/service_management/service_level_objectives/metric_slo_side_panel.png
Comment (estherk15) [inline static/images/service_management/service_level_objectives/metric_slo_side_panel.png:1]: Can you blur your name?

File: content/en/service_management/service_level_objectives/monitor.md [hunk 1]
```diff
@@ -14,11 +14,11 @@ further_reading:
 ---
 
 ## Overview
-To build an SLO from new or existing Datadog monitors, create a monitor-based SLO.
+To build an SLO from new or existing Datadog monitors, create a monitor-based SLO. Using a monitor-based SLO, you can calculate the Service Level Indicator (SLI) by dividing the amount of time your system exhibits good behavior by the total time.
 
-Time-based data sets usually map well to monitor-based SLOs. Using a monitor-based SLO, you can calculate the Service Level Indicator (SLI) by dividing the amount of time your system exhibits good behavior by the total time.
+<div class="alert alert-info">In addition to monitor-based SLOs, Time Slice SLOs are another way to create SLOs with a time-based SLI calculation. With Time Slice SLOs, you can create an uptime SLO without going through a monitor, so you don’t have to create and maintain both a monitor and an SLO.</div>
```
Comment (estherk15) [inline content/en/service_management/service_level_objectives/monitor.md:19]:
```suggestion
<div class="alert alert-info">Time Slice SLOs are another way to create SLOs with a time-based SLI calculation. With Time Slice SLOs, you can create an uptime SLO without going through a monitor, so you don’t have to create and maintain both a monitor and an SLO.</div>
```

File: content/en/service_management/service_level_objectives/metric.md [hunk 2]
```diff
@@ -18,11 +18,11 @@ further_reading:
 
 Metric-based SLOs are useful for a count-based stream of data where you are differentiating good and bad events. A metric query uses the sum of the good events divided by the sum of total events over time to calculate a Service Level Indicator (or SLI). You can use any metric to create SLOs, including custom metrics generated from [APM spans][1], [RUM events][2], and [logs][3]. For an overview on how SLOs are configured and calculated, see the [Service Level Objective][4] page.
 
-{{< img src="service_management/service_level_objectives/metric-based-slo-example.png" alt="example metric-based SLO" >}}
+{{< img src="service_management/service_level_objectives/metric_slo_side_panel.png" alt="example metric-based SLO" >}}
 
 ## Setup
 
-On the [SLO status page][5], select **New SLO +**. Then select [**Metric**][6].
+On the [SLO status page][5], select **New SLO +**. Then select [**By Count**][6].
```
Comment (estherk15) [inline content/en/service_management/service_level_objectives/metric.md:25]:
```suggestion
On the [SLO status page][5], click **+ New SLO**. Then, select [**By Count**][6].
```

File: content/en/service_management/service_level_objectives/monitor.md [hunk 3]
```diff
@@ -31,24 +31,20 @@ Datadog monitor-based SLOs support the following monitor types:
 
 ## Setup
 
-On the [SLO status page][2], select **New SLO**.
-
-Under **Define the source**, select **Monitor Based**.
+On the [SLO status page][2], select **New SLO +**. Then select **By Monitor Uptime**.
```
Comment (estherk15) [inline content/en/service_management/service_level_objectives/monitor.md:34]:
```suggestion
On the [SLO status page][2], click **+ New SLO**. Then, select **By Monitor Uptime**.
```

File: content/en/service_management/service_level_objectives/monitor.md [hunk 4]
```diff
@@ -31,24 +31,20 @@ Datadog monitor-based SLOs support the following monitor types:
...
 
 ### Define queries
 
-In the search box, start typing the name of a monitor. A list of matching monitors appears. Click on a monitor name to add it to the source list.
-
-If you're only using a single multi alert monitor in an SLO, you can optionally select "Calculate on selected groups" and pick up to 20 groups. Group selection is not supported for SLOs that contain multiple monitors. For SLOs with multiple monitors, you can add up to 20 monitors.
+In the search box, start typing the name of a monitor. A list of matching monitors appears. Click on a monitor name to add it to the source list:
```
Comment (estherk15) [inline content/en/service_management/service_level_objectives/monitor.md:38]:
```suggestion
```suggestion
In the search box, start typing the name of a monitor to see a list of matching monitors. Click on a monitor name to add it to the source list.

**Notes**:
```
```

File: content/en/service_management/service_level_objectives/monitor.md [hunk 5]
```diff
@@ -142,3 +132,4 @@ Confirm you are using the preferred SLI type for your use case. Datadog supports
 [3]: /service_management/service_level_objectives/metric/
 [4]: /synthetics/api_tests/?tab=httptest#alert-conditions
 [5]: /service_management/service_level_objectives/#slo-status-corrections
+[6]: /service_management/service_level_objectives/time_slice/
```
Comment (estherk15) [inline content/en/service_management/service_level_objectives/monitor.md:135]: Sorry if I missed this, I didn't see the associated reference link in the docs.

**Review by estherk15 — COMMENTED** [review]
Mostly non blocking feedback, and PII for an image.

File: content/en/service_management/service_level_objectives/monitor.md (hunk 5 above)
Comment (roxanne-moslehi) [reply content/en/service_management/service_level_objectives/monitor.md:135]: @estherk15 can you clarify where/how to add the reference link?

Comment (estherk15) [reply content/en/service_management/service_level_objectives/monitor.md:138]: @roxanne-moslehi If you want to hyperlink to this page, you can add brackets around the word like `[Time Slice SLO][6]`

Comment (roxanne-moslehi) [reply content/en/service_management/service_level_objectives/monitor.md:138]: @estherk15 Oh yeah I see, I had added a hyperlink but realized it didn't work in the "alert" added at the top of the page

**Review by estherk15 — APPROVED** [review]

**Comment by roxanne-moslehi** [comment]
@estherk15 We can merge this if it looks good!

**Pull request merged**